gtsfm/common/image.py

```diff
@@ -34,6 +34,13 @@ def width(self) -> int:
         """
         return self.value_array.shape[1]
 
+    @property
+    def shape(self) -> tuple:
+        """
+        The shape of the image (H, W, C).
```

Comment:
nit: can we fit it on one line? `"""The shape of the image (H,W)."""`

Btw we don't use channels elsewhere when describing the shape, do we need it here? 
https://github.com/borglab/gtsfm/blob/581f84ab605e99302a9c6425dc67e38f82b70660/gtsfm/loader/loader_base.py#L219

Reply:
Actually I think it's fine to include channels, since it just parallels what the numpy attribute .shape field would have said.

Reply:
yeah, would not want to restrict an `Image` class to 3 channel images :)